hbase-rest/src/main/java/org/apache/hadoop/hbase/rest/Constants.java

```diff
@@ -27,6 +27,8 @@
 public interface Constants {
   // All constants in a public interface are 'public static final'
 
+  int DEFAULT_HTTP_MAX_HEADER_SIZE = 64 * 1024; // 64k
```

Comment:
Can we put this in a class that isn't IA.Public?

Reply:
Maybe in RESTServer itself?

Reply:
We can do that, I was being consistent with DEFAULT_HTTP_MAX_HEADER_SIZE from  org.apache.hadoop.hbase.thrift.Constants but if we are refactoring this later I think is fine.

Reply:
makes sense, but o.a.h.h.thrift.Constants is IA.Private. I don't know why the equivalent here is IA.Public, but 🤷